### PR TITLE
feat(integration):add support for google docs api

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

- Added configuration for  google-docs in providers.yaml
- Added google-docs to docs-v2.
- Added google-docs pages for Supported API groups in mint.json.
- Added google-docs SVG file in template-logos folder.
- Consolidate google-docs docs pages

## Issue ticket number and link
EXT-43

## Test
This provider has been used successfully in local development to connect to google-docs after authorizing using Nango.

The documentation update was reviewed using `npm run docs` and verifying on localhost.

### CODE REVIEW

File: docs-v2/integrations/all/google-docs.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+| [Perform workflows](/integrate/guides/perform-workflows-with-an-api) | ✅ |
+| [Proxy requests](/integrate/guides/proxy-requests-to-an-api) | ✅ |
+| [Receive webhooks](/integrate/guides/receive-webhooks-from-an-api) | 🚫 (time to contribute: &lt;48h) |
+
+<Tip>We can implement missing features in &lt;48h, just ask for it in the [community](https://nango.dev/slack).</Tip>
+
+## Getting started
+
+-   [Google Docs access token specs](https://cloud.google.com/iam/docs/reference/sts/rest/v1/TopLevel/token#response-body)
+-   [Google Docs OAuth scopes](https://developers.google.com/docs/api/auth#docs-scopes)
```
Comment (TBonnin) [inline docs-v2/integrations/all/google-docs.mdx:23]: Should we add a link to the API reference?

Comment (hassan254-prog) [reply docs-v2/integrations/all/google-docs.mdx:23]: @TBonnin just updated my pull request to include the Google docs API reference.

File: docs-v2/integrations/productivity.mdx [hunk 2]
```diff
@@ -16,7 +16,8 @@ sidebarTitle: Productivity
     <Card title="Exact Online" href="/integrations/all/exact-online" color="#68a063" />
     <Card title="Figjam" href="/integrations/all/figjam" color="#68a063" />
     <Card title="Figma" href="/integrations/all/figma" color="#68a063" />
-    <Card title="Google Cal" href="/integrations/all/google-calendar" color="#68a063" />
+    <Card title="Google Calender" href="/integrations/all/google-calendar" color="#68a063" />
```
Comment (bodinsamuel) [inline docs-v2/integrations/productivity.mdx]:
```suggestion
    <Card title="Google Calendar" href="/integrations/all/google-calendar" color="#68a063" />
```

**Review by bodinsamuel — APPROVED** [review]

File: docs-v2/integrations/productivity.mdx (hunk 2 above)
Comment (hassan254-prog) [reply docs-v2/integrations/productivity.mdx]: @bodinsamuel , thank you for that.

**Review by TBonnin — APPROVED** [review]

**Pull request merged**